src/pages/features/index.md

```diff
@@ -39,26 +39,35 @@ Here are examples of submitting and executing Photoshop Actions.
 In this example we applied a custom Action we created called "Graphic Design."
 ![alt image](./psactions_example.png?raw=true "Original Image")
 
-### ActionJSON
+### ActionJson
```

Comment:
ActionJson -> actionJSON?

Reply:
This I kept  `A` upper case as it is a heading?